compiler/env/FEBase_t.hpp

```diff
@@ -80,6 +80,19 @@ FEBase<Derived>::allocateCodeMemory(TR::Compilation *comp, uint32_t warmCodeSize
    return warmCode;
    }
 
+// This code does not really belong here (along with allocateRelocationData, really)
+// We should be relying on the port library to allocate memory, but this connection
+// has not yet been made, so as a quick workaround for platforms like OS X <= 10.9,
+// where MAP_ANONYMOUS is not defined, is to map MAP_ANON to MAP_ANONYMOUS ourselves
+#if !defined(MAP_ANONYMOUS)
+  #define NO_MAP_ANONYMOUS
+  #if defined(MAP_ANON)
+    #define MAP_ANONYMOUS MAP_ANON
+  #else
+    #define MAP_ANONYMOUS 0
```

Comment:
Yeah, you're probably right; that's probably a more appropriate default than to silently do nothing. I'll update with your suggestion.

Thanks @charliegracie !